spec/libraries/azure_engine_spec.rb

```diff
@@ -75,11 +75,13 @@ def generate_uri(uri, options = {})
     end
   end
 
-  module Contrib
-    module Auth
-      class DummySharedAccessSignature
-        def sign
-          'https://signed.example.com'
+  module Storage
+    module Core
+      module Auth
+        class DummySharedAccessSignature
+          def signed_uri(a, b, c)
```

Comment:
__[RuboCop]__

Naming/UncommunicativeMethodParamName: Method parameter must be at least 3 characters long.


<!-- Comment by [SideCI](https://sideci.com) -->